calicoctl/commands/resources.go

```diff
@@ -102,6 +102,18 @@ func convertToSliceOfResources(loaded interface{}) []unversioned.Resource {
 	return r
 }
 
+func convertToSliceOfResourceObjects(resources []unversioned.Resource) ([]unversioned.ResourceObject, error) {
```

Comment:
I think the method convertToSliceOfResources should actually be convertToSliceOfResourceObjects ... I think that was it's intention - so shouldn't need this separate function.